### ISSUE TITLE
Multiplayer Exit Corridors in Singleplayer?

### ISSUE THREAD

**Issue by JavierNovellon** [BEEmod/BEE2.4#1154]
I was wondering if there's any possibility that Multiplayer exit corridors could be selected as Singleplayer. It does not make a lot of sense why MP maps should have different corridors than SP, plus, this would add a nice variety of corridors to choose from, beyond what's already seen countless times.

I understand that this may be impossible to do (hard-coded into the game, is that the right term?) but if it isn't, I don't see any reason why it shouldn't be done.

An example that comes to mind is in the 1970s Style, where SP Entry Corridor 4 (the stairs) would go really nicely with MP Exit Corridor 1 (also some stairs).

Is this a thing that could be done?

**Comment by LautaroL20** [comment]
LOL.

**Comment by JavierNovellon** [comment]
@LautaroL20 Listen, the question might be stupid to someone like you. But writing "LOL" does not help, nor answer the question. If you know it's not possible, then you can politely say it. "Well, this isn't a thing that can be done" That's all I need.
I don't know how either BEEmod or Portal 2 handle map generation, which is precisely why I ask.
Again though, I'm just an 18 year old sitting at home playing some games. I shouldn't be teaching you (or anybody else, for that matter) how to comment on the internet.

**Comment by TeamSpen210** [comment]
I probably don't want to directly add the ability to use the "wrong" corridor - they do have slightly different logic to account for the different number of players. But it'd be pretty trivial to copy the sh

**Comment by TeamSpen210** [comment]
I probably don't want to directly add the ability to use the "wrong" corridor - they do have slightly different logic to account for the different number of players. But it'd be pretty trivial to copy the shape of any of them over. But what I really need to do is remake most of them, a lot are really the Clean corridors with textures changed.

**Comment by King0191** [comment]
I must agree with Jaiver here. This would be really neat, and allow more variety. 

While i do not doubt you would want to add more corridors for entrance and exit that way there's less reoccurrence of corridors, anyways. I think allowing corridors from MP (like the Turret exit in P1) to be used in SP and vise-versa. Would fit in nicely and allow more customization for a map potentially around a certain theme (i.e. Turrets in P1)
